src/commands/run.js

```diff
@@ -72,8 +72,8 @@ exports.builder = function (yargs) {
   }).option('http', {
     description: 'URL for where your app is served from e.g. localhost:1234',
     default: null,
-  }).option('served-at', {
-    description: 'URL for where your app is served from e.g. localhost:1234',
+  }).option('http-served-from', {
```

Comment:
Changed the flag `--served-at` to `--http-served-from` so it is more descriptive to what the flag does.